Firestore/Source/Util/FSTUsageValidation.h

```diff
@@ -42,4 +46,8 @@ NSException *FSTInvalidUsage(NSString *exceptionName, NSString *format, ...);
     @throw FSTInvalidUsage(@"FIRInvalidArgumentException", format, ##__VA_ARGS__); \
   } while (0)
 
+#if __cplusplus
+}
```

Comment:
Should we comment what this is matching?
```
}  // extern "C"
```

(we do that in Util/FSTLogger.h, our one existing usage of extern "C")

Reply:
Done.